src/cephadm/cephadm

```diff
@@ -2650,7 +2650,10 @@ def command_bootstrap():
 
         host = get_hostname()
         logger.info('Adding host %s...' % host)
-        cli(['orch', 'host', 'add', host])
+        try:
+            cli(['orch', 'host', 'add', host])
+        except RuntimeError as e:
+            raise Error('Failed to add host: %s' % (host))
```

Comment:
Always is interesting to have information about why the command failed.. please include the error message:
`raise Error('failed to add hostn  <%s>: %s' % (host,e))`